site/content/docs/user/configuration.md

```diff
@@ -218,3 +218,54 @@ You may also want to see the [Ingress Guide].
 
 
 [Ingress Guide]: ./../ingress
+
+### Kubeadm Config Patches
+
+KIND uses [`kubeadm`](./../../design/principles/#leverage-existing-tooling) 
+to configure cluster nodes.
+Formally  KIND runs `kubeadm init` on the first control-plane node 
+which can be customized by using the kubeadm
+[InitConfiguration](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file) 
+([spec](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#InitConfiguration))
```

Comment:
there is a good chance that kubeadm v1beta1 will be removed in 1.20, so it's is better to recommend v1beta2 at this point.